addon/components/x-toggle/template.hbs

```diff
@@ -1,19 +1,26 @@
-{{#if showLabels}}
-	<span class="toggle-text toggle-prefix {{size}}">
-		{{offLabelValue}}
-	</span>
-{{/if}}
+<span class="x-toggle-component">
+  {{#if showLabels}}
+  	<span class="toggle-text toggle-prefix off-label" onclick={{action 'setToValue' false}}>
+  		{{_offLabel}}
+  	</span>
+  {{/if}}
+  <span
+    id={{elementId}}
+    class="x-toggle-container {{size}}{{if toggled ' x-toggle-container-checked'}}{{if disabled ' x-toggle-container-disabled'}}"
+    onclick={{action 'onClick'}}
+  >
+    {{input id=forId type='checkbox' checked=toggled class='x-toggle' disabled=disabled name=name}}
```

Comment:
@ksnyde good idea on converting to integration tests. My concern with the event handler and testing was to make sure that calling `$('.elementClass input').click()` would still trigger the action. If it does then it's fine. I'm not too sure whats going on with the test issue your describing but I can try taking a look when I get home tonight.